src/Microsoft.VisualStudio.ProjectSystem.Managed/Threading/Tasks/CancellationTokenExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+
+namespace Microsoft.VisualStudio.Threading.Tasks
+{
+    /// <summary>
+    ///     Provides static extensions methods for <see cref="CancellationToken"/> instances.
+    /// </summary>
+    internal static class CancellationTokenExtensions
+    {
+#pragma warning disable CA1068 // CancellationToken parameters must come last
```

Comment:
Consider filing a bug to not fire this if the `CancellationToken` is a `this` parameter to an extension method.

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/1491